OpenUtau/ViewModels/NotesViewModel.cs

```diff
@@ -91,7 +91,7 @@ public class NotesViewModel : ViewModelBase, ICmdSubscriber {
         private readonly ObservableAsPropertyHelper<double> smallChangeX;
         private readonly ObservableAsPropertyHelper<double> smallChangeY;
 
-        public readonly List<UNote> SelectedNotes = new List<UNote>();
+        public readonly NoteSelectionViewModel Selection = new NoteSelectionViewModel();
         private readonly HashSet<UNote> TempSelectedNotes = new HashSet<UNote>();
```

Comment:
Shouldn't the new class also manage temporary selection?

Reply:
Updated, now handles temp selection